codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/test-async-client-class.java

```diff
@@ -82,14 +90,14 @@ protected DefaultJsonAsyncClient(AsyncClientConfiguration clientConfiguration) {
     public CompletableFuture<APostOperationResponse> aPostOperation(APostOperationRequest aPostOperationRequest) {
 
         HttpResponseHandler<APostOperationResponse> responseHandler = protocolFactory.createResponseHandler(
-                new JsonOperationMetadata().withPayloadJson(true).withHasStreamingSuccessResponse(false),
```

Comment:
Why the changes in formatting throughout this file?

Reply:
I copied the expected file when the unit test failed. Then checked if new code looks as expected. I guess the generated file in unit test has different formatting. I tried to auto-format the file now which is having lot more changes